Core/PaymentMethodFactory.php

```diff
@@ -27,6 +28,25 @@
  */
 class PaymentMethodFactory
 {
+    /**
+     * Returns an array of payment method class names.
+     *
+     * @return array
+     *
+     * @since 1.2.0
+     */
+    public static function getPaymentMethodClasses()
+    {
+        return [
+            PaypalPaymentMethod::class,
```

Comment:
Is the order of payment methods relevant?

If not, let's please sort it **alphabetically** instead (except Credit Card that is the first).